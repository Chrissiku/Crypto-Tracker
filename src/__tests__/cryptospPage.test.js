import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import TestRenderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../Redux/configureStore';
import CryptoPage from '../Components/CryptosPage';

it('Test if header render correctly', () => {
  const TREE = TestRenderer.create(
    <Provider store={store}>
      <BrowserRouter>
        <CryptoPage />
      </BrowserRouter>
    </Provider>,
  );
  expect(TREE).toMatchSnapshot();
});
