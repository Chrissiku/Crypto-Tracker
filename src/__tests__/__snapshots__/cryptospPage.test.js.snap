// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test if The CryptoPage render correctly 1`] = `
<div
  className="crypto-container"
>
  <input
    className="search-bar"
    onChange={[Function]}
    placeholder="Search by Crypto name ..."
    type="text"
    value=""
  />
  <div
    className="card-container"
  />
</div>
`;
